server/views/components/wip/checkedItemsField.njk

```diff
@@ -0,0 +1,68 @@
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% macro renderCheckedItemsField(fieldName, field, options, values, errors) %}
```

Comment:
Why the complicated name? This is rendering a field no? I think it should be `renderField()` unless I'm missing something (and yes it checks if the field is one of the ones we support but otherwise it's rendering a field no?)

Reply:
there are 5 field templates now and they need to be imported into one template, if they all have the same name that wouldn't work. i prefer things to have 1 official name and for imports to not rename things. (similar to how we both try to avoid default exports in typescript)

Reply:
I may be a bit slow here but the names are currently confusing:
- template imports `renderFields()` from `components/wip/fields.njk` 🙃 
- `fields.njk` contains `renderFields()`, uses `renderField()`
-  `field.njk` contains `renderField()`
- `field.njk` uses `renderCheckedItemsField()` which renders either checkboxes or radios 🙃 

May I suggest perhaps we keep filenames/macro names aligned? I'm happy to stick with the `renderThing()` convention here, but I'd also name the files `renderThing.njk` so it's predictable to find those (e.g. when jumping around in the editor or otherwise).

The `renderCheckedItemsField()` confused me, specifically the "checked" part. Maybe a better name would be `renderRadiosOrCheckboxes()`?  🤔 (and yes both have items which is why they're rendered by the same macro)

Reply:
mind if i do the template/macro renaming in a future pr? cos i have 2 more lined up and dread the idea of rebasing 🤣

Reply:
hopefully this is better?
https://github.com/ministryofjustice/hmpps-incident-reporting/pull/239/commits/a29b558e5f202901e47491ec4ba95f5ab8dde9cc